server/src/packages/pas-form/bbls/bbls.controller.ts

```diff
@@ -9,6 +9,9 @@ export const BBL_ATTRIBUTES = [
   'dcp_partiallot',
   'dcp_developmentsite',
   'dcp_bblnumber',
+  'dcp_userinputborough',
```

Comment:
If these user input attributes don't exist on the database, do they just exist as local variables in the backend? When the bbl entity table gets patched, do these values just disappear since they're not saved to the db?

Reply:
@trbmcginnis these do exist as real attrs in CRM

the front end is responsible for filling out these particular fields